.env-devel

```diff
@@ -214,7 +214,7 @@ DIRECTOR_PORT=8080
 
 DIRECTOR_V2_HOST=director-v2
 DIRECTOR_V2_PORT=8000
-DIRECTOR_V2_DYNAMIC_SCHEDULER_IGNORE_SERVICES_SHUTDOWN_WHEN_CREDITS_LIMIT_REACHED=1
+DIRECTOR_V2_DYNAMIC_SCHEDULER_CLOSE_SERVICES_VIA_FRONTEND_WHEN_CREDITS_LIMIT_REACHED=1
```

Comment:
This is a new record :-)

Reply:
🥇 I deserve one of these right, right? :-)